Automattic/wp-calypso
# Update title of the plan details page

### Changes proposed in this Pull Request

Update the case of "Backup" in the title (matching the copydeck) and wrap the title in a `translate` call.

### Testing instructions

- Visit the _Plans_ section with the offer reset flow enabled
- Click on the button of the Jetpack Backup card, and check that you land in the details page
- Check the title

### Screenshots

Updated title
<img width="1082" alt="Screen Shot 2020-08-20 at 2 10 39 PM" src="https://user-images.githubusercontent.com/1620183/90810545-22610200-e2f1-11ea-8fdc-593228f97542.png">
